src/avfoundation.cs

```diff
@@ -7338,6 +7850,35 @@ interface AVVideoCompositionCoreAnimationTool {
 		AVVideoCompositionCoreAnimationTool FromComposedVideoFrames (CALayer [] videoLayers, CALayer inAnimationlayer);
 	}
 
+	[TV (11,0), NoWatch, iOS (11,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface AVCameraCalibrationData
+	{
+		[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
+		[Export ("intrinsicMatrix")]
+		Matrix3 GetIntrinsicMatrix ();
+
+		[Export ("intrinsicMatrixReferenceDimensions")]
+		CGSize IntrinsicMatrixReferenceDimensions { get; }
+
+		[MarshalDirective (NativePrefix = "xamarin_simd__", Library = "__Internal")]
+		[Export ("extrinsicMatrix")]
+		Matrix4 GetExtrinsicMatrix (); // should be a matrix 4x3
```

Comment:
This needs a new Matrix4x3 type. Please comment this code out and add a bug about it until we can fix it (it's probably easiest to fix it together with bug #[58599](https://bugzilla.xamarin.com/show_bug.cgi?id=58599).